pkg/cloudprovider/providers/gce/gce.go

```diff
@@ -134,9 +134,9 @@ type Disks interface {
 	// DiskIsAttached checks if a disk is attached to the node with the specified NodeName.
 	DiskIsAttached(diskName string, nodeName types.NodeName) (bool, error)
 
-	// DisksAreAttached is a batch function to check if a list of disks are attached
-	// to the node with the specified NodeName.
-	DisksAreAttached(diskNames []string, nodeName types.NodeName) (map[string]bool, error)
+	// DisksAreAttached is a batch function to check if disks specified in map are still
+	// attached to their respective nodes.
+	DisksAreAttached(nodeDisks map[types.NodeName][]string) (map[types.NodeName]map[string]bool, error)
```

Comment:
sorry I meant, not argument but second key. `nodeDisks map[types.NodeName][]string /*volumeId*/` sometihng  like that?